### ISSUE TITLE
[input][input-amount] Relationship with currency

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#166]
The currency of an input-amount is added as an "after". There is no programmatic relationship between the input and the "after", while there is visually. Either a builtin solution or an instruction like "add visually hidden information to the label" should be added.

For example, the label could read:  Amount (in euros). With the part in parenthesis visually hidden.

**Comment by tlouisse** [comment]
This was fixed by https://github.com/ing-bank/lion/commit/0d687bbe1e953b2b559737b9825cb055a2e9c75a#diff-89d83a2b790350b38f42f92fce0e9ff7
